smdebug/tensorflow/keras.py

```diff
@@ -69,8 +70,9 @@ def _is_not_supported(self):
             if tf.executing_eagerly() or (
                 hasattr(self.model, "run_eagerly") and self.model.run_eagerly
             ):
-                self.logger.info("Disabling SMDebug as it does not support eager mode")
-                self._hook_supported = False
+                # self.logger.info("Disabling SMDebug as it does not support eager mode")
+                # self._hook_supported = False
+                pass
```

Comment:
Why are we deleting? And if we do, these lines should be deleted rather than commented out.